dspace-api/src/main/java/org/dspace/app/harvest/HarvestCli.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.harvest;
+
+import java.sql.SQLException;
+
+import org.apache.commons.cli.ParseException;
+import org.dspace.core.Context;
+import org.dspace.eperson.EPerson;
+
+/**
+ * Test class for harvested collections.
+ *
+ * @author Alexey Maslov
```

Comment:
Silly thing, but it stuck out immediately.  Is this author tag correct?  If not, either update it or just remove it, as we don't require author tags